### DISCUSSION TITLE
Migration netbox v3.6.9 to v4.0.5 failed

### DISCUSSION THREAD

**Discussion by jaimerletona** [netbox-community/netbox#16534] [Help Wanted!]
Hello everybody.
I recently migrate from v3.6.4 to v3.6.9 perfectly fine on ubuntu 22.04 using https://netbox.readthedocs.io/en/stable/installation/upgrading/  but when I try to migrate v3.6.9 to v4.0.5 something was wrong. I am looking for help.

My requirements:
PYTHON: 
Python 3.10.12

POSTGRES:
dpkg -l | grep postgres
ii  postgresql                     14+238                                  all          object-relational SQL database (supported version)
ii  postgresql-12                  12.18-0ubuntu0.20.04.1                  amd64        object-relational SQL database, version 12 server
ii  postgresql-14                  14.12-0ubuntu0.22.04.1                  amd64        The World's Most Advanced Open Source Relational Database
ii  postgresql-client-12           12.18-0ubuntu0.20.04.1                  amd64        front-end programs for PostgreSQL 12
ii  postgresql-client-14           14.12-0ubuntu0.22.04.1                  amd64        front-end programs for PostgreSQL 14
ii  postgresql-client-common       238                                     all          manager for multiple PostgreSQL client versions
ii  postgresql-common              238                                     all          PostgreSQL database-cluster manager

REDIS:
dpkg -l | grep redis   
ii  redis-server                   5:6.0.16-1ubuntu1                       amd64        Persistent key-value database with network interface
ii  redis-tools                    5:6.0.16-1ubuntu1                       amd64        Persistent key-value database with network interface (client)

I attached
[netbox_log.txt](https://github.com/user-attachments/files/15811004/netbox_log.txt)
 a log show step by step my procedure. If any of you have an idea or an advice I appreciate it.

Regards
[netbox_log.txt](https://github.com/user-attachments/files/15811018/netbox_log.txt)

**Comment by candlerb** [comment]
> but when I try to migrate v3.6.9 to v4.0.5 something was wrong

An upgrade directly from v3.6 to v4.0 is not permitted. Because this is a major version upgrade, you first have to upgrade to the previous minor (v3.7.x)

**Reply by jaimerletona** [reply]
Thanks for orientation.
I tried upgrade v3.6.9 to v.3.7.8 and I obtain this errors:

![image](https://github.com/netbox-community/netbox/assets/36545400/41ecd8ce-9bea-4ea3-ad26-76d826c01420)

Any ideas or orientation?
Thank you in advance.
